File: Dart/lib/canAliceWin.dart
class Solution {
  bool canAliceWin(List<int> nums) {
    var sum = 0;
    var singleDigitSum = 0;
    for (var n in nums) {
      sum += n;
      if (n < 10) {
        singleDigitSum += n;
      }
    }
    return sum != (singleDigitSum * 2);
  }
}

File: C/exercise/canAliceWin.c

#include "canAliceWin.h"

bool canAliceWin(int *nums, int numsSize)
{
	int sum = 0;
	for (int i = 0; i < numsSize; i++) {
		sum += (nums[i] < 10) ? nums[i] : -nums[i];
	}
	return sum != 0;
}
